src/webui/btjson.cpp

```diff
@@ -708,6 +711,7 @@ QVariantMap getTranserInfoMap()
     map[KEY_TRANSFER_AVERAGE_TIME_QUEUE] = cacheStatus.averageJobTime;
     map[KEY_TRANSFER_TOTAL_QUEUED_SIZE] = cacheStatus.queuedBytes;
 
+    map[KEY_TRANSFER_FREESPACEONDISK] = Utils::Fs::freeDiskSpaceOnPath(BitTorrent::Session::instance()->defaultSavePath());
```

Comment:
I didn't work with threads in Qt before, Can you guide me which way is suitable in this situation?
I read about `QtConcurrent::run()` but I don't know how to use it here in this situation with WebUI refresh.